src/components/Books.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Books = () => (
+  <>
+    <div className="books-container" />
+    <h2>ADD NEW BOOK</h2>
+    <form>
+      <input type="text" placeholder="Book title" />
+      <select disabled>
+        <option value="category">Category</option>
+      </select>
+      <button type="button">ADD BOOK</button>
+    </form>
```

Comment:
- [ ] Please move your form to a separate component,  for example, `CreateNewBook` 😄